### PR TITLE
Ddp 7383 create os somatic assent addendum

### PR DESCRIPTION
**Pull request by nsakande**
https://broadinstitute.atlassian.net/jira/software/c/projects/DDP/boards/836?modal=detail&selectedIssue=DDP-7383&sprint=1518

### CODE REVIEW

File: study-builder/studies/pancan/i18n/en.conf [hunk 1]
```diff
@@ -466,7 +466,7 @@
...
     "s1_participation_body2": "We will ask that you send a sample of your child’s saliva to us in a pre-stamped package that we will provide.",
     "s1_participation_body3": "We will ask you to complete a medical release form so that we are able to request your child’s medical records. If we do so, we will take care of obtaining copies of your child’s records from the hospitals or centers where your child receives their medical care. If needed, we may contact you to ask if you would be willing to sign additional authorization forms or documents that certain hospitals or centers may require in order to share copies of your child’s medical records with us.",
-    "s1_participation_body4": "If you choose to share tissue specimens with us, we may also obtain small amounts of your child’s stored cancer samples from hospitals or centers where your child receives their care.",
+    "s1_participation_body4": "If you choose to s"The Osteosarcoma Project" is a patient-driven movement that empowershare tissue specimens with us, we may also obtain small amounts of your child’s stored cancer samples from hospitals or centers where your child receives their care.",
```
Comment (archilsharashenidzebroad) [inline study-builder/studies/pancan/i18n/en.conf:469]: check if this variable is used somewhere else, to make sure your patch is not updating other place accidentally

File: study-builder/studies/osteo/parental-consent.conf [hunk 2]
```diff
@@ -209,8 +209,8 @@
           "blockType": "GROUP",
```
Comment (archilsharashenidzebroad) [inline study-builder/studies/osteo/parental-consent.conf:209]: This kind of changes sometimes make it hard to review PR. try not to prettify already existing code if it's not crucial.

File: study-builder/studies/osteo/consent-assent.conf [hunk 3]
```diff
@@ -210,8 +210,8 @@
           "blockType": "GROUP",
           "shownExpr": null
         },
-        { include required("snippets/consent-minors-section3-additional-terms.conf") },
```
Comment (archilsharashenidzebroad) [inline study-builder/studies/osteo/consent-assent.conf:213]: This kind of changes sometimes make it hard to review PR. try not to prettify already existing code if it's not crucial.

**Review by archilsharashenidzebroad — CHANGES_REQUESTED** [review]

File: study-builder/studies/pancan/i18n/en.conf (hunk 1 above)
Comment (nsakande) [reply study-builder/studies/pancan/i18n/en.conf:469]: This variable is not used other place.

File: study-builder/studies/osteo/parental-consent.conf (hunk 2 above)
Comment (nsakande) [reply study-builder/studies/osteo/parental-consent.conf:209]: Okay

**Review by archilsharashenidzebroad — APPROVED** [review]
LGTM

**Pull request merged**